languages/ruby/exercises/concept/numbers/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+### General
+
+- [The Ruby `Float` class][float-ruby]
```

Comment:
I'd be slightly hesitant to point to the official Ruby docs as they can be quite overwhelming. My preference would be to link to specific methods in the task-specific hints (without giving away the actual name of the method in this file).